pokemongo_bot/cell_workers/catch_lured_pokemon_worker.py

```diff
@@ -30,14 +30,10 @@ def get_lured_pokemon(self):
             return False
 
         fort = forts[0]
-
-        self.api.fort_details(fort_id=fort['id'],
+        details = fort_details(self.bot, fort_id=fort['id'],
                               latitude=fort['latitude'],
                               longitude=fort['longitude'])
-
-        response_dict = self.api.call()
-        fort_details = response_dict.get('responses', {}).get('FORT_DETAILS', {})
-        fort_name = fort_details.get('name', 'Unknown').encode('utf8', 'replace')
+        fort_name = details.get('name', 'Unknown').encode('utf8', 'replace')
```

Comment:
You could probably use this same line in seen_fort_worker. You have a try catch there right now.